test/envs/volsync.yaml

```diff
@@ -26,6 +28,7 @@ templates:
     addons:
       - volumesnapshots
       - csi-hostpath-driver
+      - metrics-server
```

Comment:
Why do need this in the volsync env? it is used only for testing volsync.

Reply:
I was trying to keep the minikube addons list the same everywhere where we use VMs.

Reply:
Lets keep the minimal testing environments minimal and modify only {regional,metro}-dr.yaml.